pages/index.js

```diff
@@ -71,6 +71,7 @@ export default function HomePage({ projects, favorites, onToggleFavorite, theme
         projectsToDisplay={projectsToDisplay}
         favorites={favorites}
         onToggleFavorite={onToggleFavorite}
+        isWhite={true} 
```

Comment:
![What](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExcXRwZm5oamkxbmZwbDAwN2hyOGoyemlrOGk0N3A5Mmloam83NnUweiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/YVPwi7L2izTJS/giphy.gif)

Reply:
I try to explain it, 😆
"isWhite" is for the Short-fact-boxes in the homepage, 
the color for the category "fashion" clashes with the color of the Short-fact-boxes,
so we made Short-fact-boxes white but only in the homepage.